.github/workflows/test.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
       matrix: ${{ fromJson(needs.setup.outputs.matrix) }}
     defaults:
       run:
-        shell: bash -l {0}
+        shell: bash -e {0}
```

Comment:
There is no need to load the settings file (`-l`), like `.bashrc` which could pollute the PATH. 

Though it is important to exit if one command fails (`-e`).